scripts/validate

```diff
@@ -13,10 +13,10 @@ function chsv_check_version() {
 }
 
 echo "==> Checking if pkg directory contains only semver compatible version"
-
+const="20.10.latest"
 for DOCKER_VERSION in $(ls -1 ../pkg/); do
     SEMVER=$(chsv_check_version $DOCKER_VERSION)
-    if [ -z $SEMVER ]; then
+    if [ -z $SEMVER ] && [ -z $const ]; then
```

Comment:
I guess this needs to be `$DOCKER_VERSION ~= $const` or something, but I don't get what `20.10.latest` is as that is not present in this PR.

Reply:
ah! right. earlier I was testing if `20.10.latest` works or not, later I changed that but seems like this got left out. This should be `20.10`. Will update this.